modules/dnn/src/tensorflow/tf_importer.cpp

```diff
@@ -2727,6 +2730,10 @@ void TFImporter::populateNet()
     addConstNodes(netBin, value_id, layers_to_ignore);
     addConstNodes(netTxt, value_id, layers_to_ignore);
 
+    if (DNN_DIAGNOSTICS_RUN)
+    {
+        LayerFactory::registerLayer("NotImplemented", notImplementedRegisterer);
+    }
```

Comment:
We have other importers too with similar problems.
What would happen with multiple registration attempts?

IMHO, register should go into `NotImplemented` as static method.